js/w3c/headers.js

```diff
@@ -296,6 +296,9 @@ define(
                 conf.publishHumanDate = utils.humanDate(conf.publishDate);
                 conf.isNoTrack = $.inArray(conf.specStatus, this.noTrackStatus) >= 0;
                 conf.isRecTrack = conf.noRecTrack ? false : $.inArray(conf.specStatus, this.recTrackStatus) >= 0;
+                conf.isMemberSubmission = conf.specStatus === "Member-SUBM";
```

Comment:
This looks good, but I noticed that in the status2rdf and status2maturity mapping tables elsewhere in this document there is no mapping for these two types of document.  The rdf will require an update to  http://www.w3.org/2001/02pd/rec54, which may not be in the offing right now.  But what maturity should these be classified as?  A special class (SUBMISSION)?  Or "NOTE"?